common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -311,6 +318,37 @@ static void copyMemory(Object src, long srcOffset, Object dst, long dstOffset, l
         }
     }
 
+    static boolean equals(final byte[] bytes1, final byte[] bytes2) {
+        if (bytes1.length != bytes2.length) {
+            return false;
+        }
+        if (bytes1.length == 0) {
+            return true;
+        }
+        int remainingBytes = bytes1.length & 7;
+        for (int i = bytes1.length - 8; i >= remainingBytes; i -= 8) {
+            final long word1 = UNSAFE.getLong(bytes1, ARRAY_BASE_OFFSET + i);
```

Comment:
Agreed that it may be a small thing but if we are trying to communicate conceptual meaning of values with local variable then can we use a comment?  Comments are 0 work for JIT :)  Things are pretty self contained here and the purpose for this method is to optimize an operation.  BTW 169 lines of byte code with inline suggestions vs 192 as is without any changes.